docs/website/docs/getting-started/index.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started Guide
+
+## Setup
+Setup your local environment by installing IREE's prebuilt binaries. See the comprehensive
+installation [guide](../bindings/python.md) for installation of individual components, the default installation is below.
+
+```
+python -m pip install \
+  iree-compiler \
+  iree-runtime \
+  iree-tools-tf \
+  iree-tools-tflite \
+  iree-tools-xla
+```
+
+## Supported frameworks
+
+See end-to-end examples of how to execute a variety models on IREE. This covers
+the import, compilation, and execution of the provided model.
+
+* [TensorFlow](./tensorflow.md)
+* [TensorFlow Lite](./tflite.md)
+* [JAX](./jax.md)
+
+Importing from PyTorch and other frameworks is planned - stay tuned!
+
+## Samples
+
+Check out the samples in IREE's
+[colab/ directory](https://github.com/google/iree/tree/main/colab) and the
```

Comment:
Check out the samples in IREE's colab/ directory, as well as the iree-samples respository, which contains workflow comparisons across frameworks.

Reply:
Done.